.github/workflows/tests-polkadot-deps.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x]
```

Comment:
While we’re at it, should we add 20 as well?

Reply:
yeah why not, maybe as an optional?